src/despacer.c

```diff
@@ -1111,6 +1122,32 @@ size_t avx2_countspaces(const char *bytes, size_t howmany) {
   return count;
 }
 
+bool avx2_hasspace(const char *bytes, size_t howmany) {
+  __m256i spaces = _mm256_set1_epi8(' ');
+  __m256i newline = _mm256_set1_epi8('\n');
+  __m256i carriage = _mm256_set1_epi8('\r');
+  size_t i = 0;
+  for (; i + 31 < howmany; i += 32) {
+    __m256i x = _mm256_loadu_si256((const __m256i *)(bytes + i));
+    // we do it the naive way, could be smarter?
+    __m256i xspaces = _mm256_cmpeq_epi8(x, spaces);
+    __m256i xnewline = _mm256_cmpeq_epi8(x, newline);
+    __m256i xcarriage = _mm256_cmpeq_epi8(x, carriage);
+    __m256i anywhite =
+        _mm256_or_si256(_mm256_or_si256(xspaces, xnewline), xcarriage);
```

Comment:
The identation of this code around here seems somewhat random. Code is easier to read when indentation follows a consistent model.

Reply:
Fixed it.